extensions/stackdriver/log/logger.cc

```diff
@@ -87,20 +97,55 @@ void Logger::addLogEntry(const ::Wasm::Common::RequestInfo& request_info,
           request_info.start_time);
   new_entry->set_severity(::google::logging::type::INFO);
   auto label_map = new_entry->mutable_labels();
+  (*label_map)["request_id"] = request_info.request_id;
   (*label_map)["source_name"] = peer_node_info.name();
   (*label_map)["source_workload"] = peer_node_info.workload_name();
   (*label_map)["source_namespace"] = peer_node_info.namespace_();
+  // Add source app and version label if exist.
+  const auto& peer_labels = peer_node_info.labels();
+  auto version_iter = peer_labels.find("version");
+  if (version_iter != peer_labels.end()) {
+    (*label_map)["source_version"] = version_iter->second;
+  }
+  auto app_iter = peer_labels.find("app");
+  if (app_iter != peer_labels.end()) {
+    (*label_map)["source_app"] = app_iter->second;
+  }
 
-  (*label_map)["request_operation"] = request_info.request_operation;
   (*label_map)["destination_service_host"] =
       request_info.destination_service_host;
   (*label_map)["response_flag"] = request_info.response_flag;
-  (*label_map)["protocol"] = request_info.request_protocol;
```

Comment:
I think we should keep this?

Reply:
I moved it to http request. Since it already exists there and I think it makes sense to just use that field.